tests/data/test_data_preprocessing_utils.py

```diff
@@ -781,7 +1110,13 @@ def test_process_dataconfig_multiple_files_varied_data_formats(
 
     tokenizer = AutoTokenizer.from_pretrained(MODEL_NAME)
     with pytest.raises(
-        (AssertionError, datasets.exceptions.DatasetGenerationCastError)
+        (
+            AssertionError,
+            ValueError,
+            datasets.exceptions.DatasetGenerationCastError,
+            pyarrow.lib.ArrowInvalid,
```

Comment:
in which case is pyarrow error thrown?

Reply:
I removed this unit test as `test_process_dataconfig_multiple_files_varied_data_formats` will not throw error now because we are removing validate_dataset function and hence datasets will any columns (varied format) will get concatenated and will not give error. Though e2e successful run with varied format depends on which handler user passes. @dushyantbehl 

CC: @willmj